spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -52,6 +52,7 @@ object StandardizationJob {
   private val log = LoggerFactory.getLogger(this.getClass)
   private val conf = ConfigFactory.load()
   private val menasBaseUrls = MenasConnectionStringParser.parse(conf.getString("menas.rest.uri"))
+  private val controlInfoValidation = conf.getString("control.info.validation")
```

Comment:
Maybe this would deserve to get extracted somewhere (perhaps into `ControlInfoValidation`) -- given the fact that it is shared with Conformance?